azureFunction/function.json

```diff
@@ -0,0 +1,21 @@
+{
+  "bindings": [
+    {
+      "type": "eventHubTrigger",
+      "name": "eventHubMessages",
+      "direction": "in",
+      "path": "%eventHubConnectionPath%",
+      "connection": "eventHubConnectionString"
+    },
+    {
+      "name": "$return",
+      "type": "sendGrid",
+      "direction": "out",
+      "apiKey": "mySendGridKey",

```

Comment:
For those field that users need to provide, make the format like `"apiKey": "<mySendGridKey>"`, `"to": "<abcde>"`